profile/README.md

```diff
@@ -0,0 +1,89 @@
+
+
+### Stargazing
+- 26,105 ⭐️ [bubbletea](https://github.com/charmbracelet/bubbletea) - A powerful little TUI framework 🏗 [_v0.12.1_](https://github.com/charmbracelet/bubbletea/releases/tag/v0.12.1)
+- 17,604 ⭐️ [gum](https://github.com/charmbracelet/gum) - A tool for glamorous shell scripts 🎀 [_v0.1.0_](https://github.com/charmbracelet/gum/releases/tag/v0.1.0)
+- 15,580 ⭐️ [glow](https://github.com/charmbracelet/glow) - Render markdown on the CLI, with pizzazz! 💅🏻 [_v0.1_](https://github.com/charmbracelet/glow/releases/tag/v0.1)
+- 14,621 ⭐️ [vhs](https://github.com/charmbracelet/vhs) - Your CLI home video recorder 📼 [_v0.0.1_](https://github.com/charmbracelet/vhs/releases/tag/v0.0.1)
```

Comment:
huh, these releases are showing up in the wrong order (oldest first)... Will need to fix this in markscribe